.flake8

```diff
@@ -1,2 +1,7 @@
 [flake8]
+exclude = .eggs,build,docs,.venv*
 ignore = E203 E501 W503 W504
+per-file-ignores =
+    praw/models/__init__.py:F401
+    praw/models/listing/mixins/__init__.py:F401
+    praw/models/reddit/mixins/__init__.py:F401
```

Comment:
careful about newline at EOF

Reply:
It's fine.

Reply:
FYI, there's a built-in hook to fix this problem automatically, `end-of-file-fixer`. You might be interested in some of the other out of the box [pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks) as well; they are very cheap to run and can fix many common problems automatically.